src/Table/helpers.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+
+export function getChidlName(child: React.Element<any>): string {
```

Comment:
is there a chance child will be `null`?

Reply:
good point, you can never know what can be null.